.github/workflows/main.yml

```diff
@@ -22,6 +22,12 @@ jobs:
             rust: beta
           - os: ubuntu-20.04
             rust: nightly
+          - os: ubuntu-24.04
+            rust: stable
+          - os: ubuntu-24.04
+            rust: beta
+          - os: ubuntu-24.04
+            rust: nightly
```

Comment:
hmm, why are we bumping this? I kinda think we should just bump ahead to a recent ubuntu if we gotta?

Reply:
--compress-debug-sections=zstd is only available in the binutils/etc that ships with 24.04, not with 20.04 or 22.04.

Reply:
And no chance of installing that stuff on ubuntu 20 without exploding the world, huh?

Reply:
I would be scared to try :)

If rust-lld propagates out to the stable branches I think it'll just work there though.

Reply:
Oh cool. Can you give that a whirl against our CI for nightly + ubuntu 20.04, then?

Reply:
``` = note: rust-lld: error: --compress-debug-sections: LLVM was not built with LLVM_ENABLE_ZSTD or did not find zstd at build time```

Ah, apparently not. Seems like it should be fixable on rustc's end though.

Reply:
Looks like that's https://github.com/rust-lang/rust/issues/120953

Reply:
huh.

Reply:
...wait, can we just install lld and test with that or something?

Reply:
https://github.com/rust-lang/rust/pull/125642 is also good lmao.

Reply:
According to rust-lang/rust#124129 we can just use `-Zlinker-features=-lld` but I feel like rust-lld should be fixed.